core/src/main/java/io/questdb/griffin/engine/table/AsyncGroupByAtom.java

```diff
@@ -233,20 +259,13 @@ public RecordSink getMapSink(int slotId) {
     }
 
     // Thread-unsafe, should be used by query owner thread only.
-    public Particle getOwnerParticle() {
-        return ownerParticle;
-    }
-
-    public Particle getParticle(int slotId) {
-        if (slotId == -1) {
-            return ownerParticle;
-        }
-        return perWorkerParticles.getQuick(slotId);
+    public MapFragment getOwnerFragment() {
```

Comment:
unused

Reply:
Removed in 6dd10b7c354e0cd9db61b549b6fc80dde78baf3e